src/components/form/select/select.vue

```diff
@@ -5,12 +5,21 @@
                 <slot name="label"/>
             </div>
             <div class="ui-form-field--input-container" :data-action-count="actionCount || 1">
+                <span class="ui-form-field--floating-label" v-if="hasFloatingLabel && $scopedSlots.label">
+                    <slot name="label"/>
```

Comment:
If you have `hasFloatingLabel` set to true, two slots with the name `label` are rendered. I think this might produce some errors.

Reply:
This code is similar to the one used for the input fields https://github.com/dreipol/vue-ui/blob/master/src/components/form/input/input.vue#L4-L10 .